inc/saiport.h

```diff
@@ -2230,6 +2322,43 @@ typedef sai_status_t (*sai_clear_port_pool_stats_fn)(
         _In_ uint32_t number_of_counters,
         _In_ const sai_stat_id_t *counter_ids);
 
+/**
+ * @brief Attribute data for #SAI_PORT_SERDES_ATTR_TX_FIR_TAP_MODE
+ *
+ * TX FIR Tap modes for NRZ and PAM4
```

Comment:
I keep getting "WARNING: Word 'NRZ' is misspelled or is acronym, add to acronyms.txt? saiport.h 2328: * TX FIR Tap modes for NRZ and PAM4"  warning.  NRZ is already listed in the acronyms file.  Why is this warning?  How do we fix it?